### PR TITLE
feat: add support for OptiX 8.

### PR DESCRIPTION
**Pull request by rabbitLove520**

### CODE REVIEW

File: SConstruct [hunk 1]
```diff
@@ -5,7 +5,7 @@ DEBUG = ARGUMENTS.get("debug", 0)
 
 CUDA_PATH = os.environ['CUDA_PATH']
 CUDA_INCLUDE_PATH = CUDA_PATH + "/include"
-OPTIX_PATH = "C:/ProgramData/NVIDIA Corporation/OptiX SDK 7.6.0"
+OPTIX_PATH = "/home/baoxiaohe/NVIDIA-OptiX-SDK-8.0.0-linux64-x86_64"
```
Comment (DeclanRussell) [inline SConstruct]: This looks like a local path on your machine right? Is there a default install directory on linux for OptiX? If not, maybe we should make this an argument for scons?

File: .vscode/c_cpp_properties.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{
```
Comment (DeclanRussell) [inline .vscode/c_cpp_properties.json]: Could you remove remove your .vscode settings from this PR? Other people maybe have their own settings when building this project.

**Review by DeclanRussell — CHANGES_REQUESTED** [review]
Thanks for the efforts on this, I've left a couple of minor comments on the PR. I've not looked at this project too much lately as Nvidia now have a pretty good command line tool as part of the OptiX SDK samples. Have you checked this out?

File: .vscode/c_cpp_properties.json (hunk 2 above)
Comment (rabbitLove520) [reply .vscode/c_cpp_properties.json]: ok

Comment (rabbitLove520) [reply .vscode/c_cpp_properties.json]: I have changed my code.

**Review by DeclanRussell — APPROVED** [review]
LGTM! Nice work. Thanks!

File: SConstruct (hunk 1 above)
Comment (rabbitLove520) [reply SConstruct]: I have added some instructions on README.md and don't change the SConstruct file.

**Comment by rabbitLove520** [comment]
> Thanks for the efforts on this, I've left a couple of minor comments on the PR. I've not looked at this project too much lately as Nvidia now have a pretty good command line tool as part of the OptiX SDK samples. Have you checked this out?

not yet, and your tool is easy to use.

**Pull request merged**